packages/ui-kit/src/components/Radio/Radio.tsx

```diff
@@ -0,0 +1,52 @@
+import { FC, ReactNode, InputHTMLAttributes } from "react";
+import clsx from "clsx";
+import { nanoid } from "nanoid";
+
+import styles from "./Radio.module.css";
+import { CheckIcon } from "../icons";
+import { Label } from "../Label";
+
+export interface RadioProps
+  extends Omit<InputHTMLAttributes<HTMLInputElement>, "className"> {
+  label?: string | ReactNode;
+  className?: {
+    container?: string;
+    inputContainer?: string;
+    input?: string;
+    radio?: string;
+    label?: string;
+  };
+}
+
+export const Radio: FC<RadioProps> = ({
```

Comment:
As for the custom looks of it, I'd either remove the radio box all together from the ui kit (and only leave the radio boxes implementation in checkout) or if it's to be used accross projects I'd make a separate component called RadioBox and RadioBoxGroup (if necessary) and keep the styling and displaying logic we currently have in checkout there. Do we know if this is checkout specific or not, @pwgryglak ?

Reply:
As of now it's only a part of checkout